pages/hydra-ecosystem-tools-overview.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Tools | Hydraecosystem.org
+permalink: /hydra-ecosystem-tools
+---
+
+# Generics
+
+## Hydrus
+
+**Hydrus** is a generic server stack that is used for easier and efficient creation of Hypermedia driven REST-APIs with the use of Hydra. Hydrus utilizes the power of Linked Data to create powerful REST APIs to serve data. Hydrus uses the Hydra(W3C) standard for creation and documentation of it’s APIs. It is a project created from scratch with minimal requirements so that anyone can use it in their project without any difficulties. Hydra is basically a python based library, which enables us to create (smart) APIs which can be interpreted by machines. In order to leverage the benefit of Hydrus library apart from dependent python libraries, we don't need much apart from an API doc. Actually Hydrus expects hydraDoc object, but when we pass an API Doc to it. It internally uses the Doc Maker tool to create a HydraDoc object for it. After receiving hydraDoc object, Hydrus parses the data into `Classes`, `Properties`, `Collections`. Which are then utilized to create smart APIs.  
+
+Diagram shown below should make things more clear. 
+
+<img src="https://gsocchrizandr.files.wordpress.com/2017/06/flo1.png"/>
+
+Hydrus supports following features-
+* A client that can understand Hydra vocabulary and interacts with a Hydra supporting server to basic CRUD operations on data.
+* A generic server that can serve required data and metadata(in the form of API documentation) to a client over HTTP.
+* A middleware that allows users to use the client to interact with the server using Natural Language which is processed machine consumable language. __(under development)__.
+
+This brief description of Hydrus must have provided a broad Idea of Hydrus. To know more, visit this [link](/hydrus). It is recommended to imagine use cases where it can be beneficial and try it own your own. You can read [Hydrus - Beginner's Guide](/getting-started-with-hydrus) in order to get started and you can find codebase [here](https://github.com/HTTP-APIs/hydrus).
+
+## Hydra Python Agent
+
+**Hydra python agent** is a smart Hydra client implemented in Python which can interact with hydrus and query data via basic CRUD operatrions. Reference implementation is [Heracles.ts](https://github.com/HydraCG/Heracles.ts). Smart clients are generic automated clients that establish resilient connected data networks leveraging knowledge graphs. Redis is used for data storage and data can be queried using OpenCypher. 
+
+**Hydra python agent** is designed to:
+* Provide a seamless Client that can be used to interact with Hydra based Smart APIs.
+* Cache metadata from the Hydra server it connects to, hence allowing querying on the client-side.
+* Maintain a synchronization mechanism that assures consistency of cached resources.
+
+### Worth Knowing
```

Comment:
This should not be in overview. You could have added this in the agent page instead of just using the README from the repo.